src/libstd/sys/windows/rand.rs

```diff
@@ -23,7 +23,7 @@ impl OsRng {
     pub fn new() -> io::Result<OsRng> {
         let mut hcp = 0;
         let ret = unsafe {
-            c::CryptAcquireContextA(&mut hcp, 0 as c::LPCSTR, 0 as c::LPCSTR,
+            c::CryptAcquireContextA(&mut hcp, ptr::null_mut(), ptr::null_mut(),
```

Comment:
My bad, my amend commits from Win PC are gone... I hope reflog can recover
them

野田  開 <nodakai@gmail.com>

2016-04-04 0:34 GMT+08:00 Oliver Middleton <notifications@github.com>:

> In src/libstd/sys/windows/rand.rs
> <https://github.com/rust-lang/rust/pull/32701#discussion_r58313013>:
>
> > @@ -23,7 +23,7 @@ impl OsRng {
> >      pub fn new() -> io::Result<OsRng> {
> >          let mut hcp = 0;
> >          let ret = unsafe {
> > -            c::CryptAcquireContextA(&mut hcp, 0 as c::LPCSTR, 0 as c::LPCSTR,
> > +            c::CryptAcquireContextA(&mut hcp, ptr::null_mut(), ptr::null_mut(),
>
> LPCSTR is a *const CHAR so I think these should be just ptr::null().
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/rust-lang/rust/pull/32701/files/4a2f6da8955784a64258f01eebc06deeba29999a#r58313013>
>